src/main/java/uk/gov/hmcts/darts/audio/service/impl/AudioTransformationServiceImpl.java

```diff
@@ -205,7 +206,13 @@ private UUID processAudioRequest(Integer requestId) {
             throw new DartsApiException(AudioApiError.FAILED_TO_PROCESS_AUDIO_REQUEST, e);
         }
 
-        mediaRequestService.updateAudioRequestStatus(mediaRequestEntity.getId(), COMPLETED);
+        AudioRequestOutputFormat audioRequestOutputFormat = AudioRequestOutputFormat.MP3;
+        if (mediaRequestEntity.getRequestType().equals(DOWNLOAD)) {
+            audioRequestOutputFormat = AudioRequestOutputFormat.ZIP;
+        }
+        String fileName = mediaRequestEntity.getHearing().getCourtCase().getCaseNumber() + "_" + mediaRequestEntity.getHearing().getHearingDate();
```

Comment:
The ticket suggests the date format in the string should be in the form `20_OCT_2023`, but string conversion of `getHearingDate`'s `LocalDate` will produce `YYYY-MM-DD` (e.g. `2023-10-20`).

I agree what you have here is better, but worth clarifying if that format is important.

Reply:
Changed date format to same as Jira ticket - can always amend it later if required 👍